tests/test_wait_for_user_rhythm.py

```diff
@@ -42,17 +43,16 @@ def _patched_sleep(self, seconds):
         self._sleeping = False
 
     def advance_patched_sleep(self, seconds: float = 0.01):
-        """ Makes _patched_sleep() return
-        Allows controlling how many times sleep() returns.
-        """
+        """ Makes _patched_sleep() return. Allows controlling how many times sleep() returns.  """
         expected_sleeps = seconds / WaitForUserRhythm.sleep_time
         for _ in range(round(expected_sleeps)):
             while not self._sleeping or self._return_from_sleep:
                 time.sleep(0.001)
             self._return_from_sleep = True
 
     def start_wait_for_bell_time_thread(self, current_time, bell, row_number, place, user_controlled, stroke):
-        """ Runs wait_for_bell_time() on a different thread, as it should loop until on_bell_ring() is called
+        """
```

Comment:
As a formatting question, is this because the doc string would be too long on a single line?

Reply:
Yes I thought this was how PEP-8/PEP-257 says docstrings should be formatted.  It turns out I'm wrong, but correcting all these is the job for another PR.  I'll revert all the ones I've changed.